helpers/controllers/submissionController.ts

```diff
@@ -11,14 +11,14 @@ import { prisma } from '../../prisma'
 const { User, Submission, Challenge, Lesson } = db
 
 type ArgsCreateSubmission = {
-  lessonId: string
+  lessonId: number
   cliToken: string
   diff: string
   challengeId: string
 }
 
 type ArgsGetSubmissions = {
-  lessonId: string
+  lessonId: number
 }
 
 export const createSubmission = async (
```

Comment:
Use the types I mentioned here https://github.com/garageScript/c0d3-app/pull/629#discussion_r612877071